op-monitorism/faultproof_withdrawals/state.go

```diff
@@ -0,0 +1,165 @@
+package faultproof_withdrawals
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/ethereum-optimism/monitorism/op-monitorism/faultproof_withdrawals/validator"
+	"github.com/ethereum-optimism/optimism/op-service/metrics"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type State struct {
+	nextL1Height   uint64
+	latestL1Height uint64
+
+	processedProvenWithdrawalsExtension1Events uint64
+
+	isDetectingForgeries uint64
```

Comment:
we should change the name of the `isDetectingForgeries` with someone more like numberish things.